kfdefs/overlays/moc/smaug/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ds-ml-workflows-ws
```

Comment:
the kustomize build is failing because in this kustomization there are directories listed which do not exist. I would suggest adding only the `thoth-infra-prod` and `opf-seldon` directories in this list since they are the ones you are adding in this PR